### ISSUE TITLE
DbCurrentSystem access not synchronized properly

### ISSUE THREAD

**Issue by ypujante** [pongasoft/glu#299]
From forum thread: http://glu.977617.n3.nabble.com/Potential-synchronization-issue-td4026912.html

**Comment by ypujante** [comment]
```
2015/05/20 16:32:17.897 ERROR [PatchedDefaultFlushEventListener] Could not synchronize database state with session 
org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect): [org.linkedin.glu.console.domain.DbCurrentSystem#1] 
        at org.grails.datastore.gorm.GormStaticApi.withTransaction(GormStaticApi.groovy:687) 
        at org.linkedin.glu.console.provisioner.services.storage.SystemStorageImpl.saveCurrentSystem(SystemStorageImpl.groovy:68) 
        at org.linkedin.glu.orchestration.engine.system.SystemServiceImpl.saveCurrentSystem(SystemServiceImpl.groovy:163) 
        at org.linkedin.glu.orchestration.engine.system.AuditedSystemService.saveCurrentSystem(AuditedSystemService.groovy:37) 
        at org.linkedin.glu.console.controllers.ModelController$_saveCurrentSystem_closure14.doCall(ModelController.groovy:191) 
        at org.linkedin.glu.console.controllers.ControllerBase.withLock(ControllerBase.groovy:119) 
        at org.linkedin.glu.console.controllers.ModelController.saveCurrentSystem(ModelController.groovy:184) 
        at org.linkedin.glu.console.controllers.ModelController$_closure9.doCall(ModelController.groovy:271) 
```

**Comment by ypujante** [comment]
Fixed in 5.6.1
